src/index.ts

```diff
@@ -46,13 +46,13 @@ module.exports = async (app: Application) => {
       logger.debug("Config exists");
       logger.debug(config);
       await handle(context, eventType!, 30000).catch(err => {
-        logger.error(err);
+        logger.warn(err);
```

Comment:
This is because I am seeing a bunch of errors like:

```
{"name":"event","hostname":"XXXXXX","pid":38,"id":"5aa48700-dcd8-11ea-9da4-144379c1db2e","owner":"XXXX","repo":"XXXX","issue":64422,"app":"probot-require-label","level":"ERROR","err":{"message":"Couldn't remove label: needs:Team for issue: 64422, error: HttpError: Label does not exist","name":"Error","stack":"Error: Couldn't remove label: needs:Team for issue: 64422, error: HttpError: Label does not exist\n    at /botelastic/node_modules/probot-require-label/lib/handler.js:126:51"},"msg":"Couldn't remove label: needs:Team for issue: 64422, error: HttpError: Label does not exist","time":"2020-08-12T20:14:01.048Z","v":0}
```

The above error is not super useful for people operating a probot instance. I wish there was a way to alert the consumers of the bot about this rather than logging it in `stdout`